pkg/core/unit32.go

```diff
@@ -0,0 +1,9 @@
+package core
+
+// slice attaches the methods of Interface to []int, sorting in increasing order.
+type slice []uint32
```

Comment:
`unit32.go` is still with us.

Reply:
renamed to `uint32`. It was not that obvious from the first time that it was a spelling issue.